docs/recipes/add_model_setup.rst

```diff
@@ -222,84 +222,122 @@ Implement a New Coupled Setup
 
 **Feature available since version:** 4.2
 
+An example of the different files needed for `AWICM` setup is included at the end of this section
+(see :ref:`recipes/add_model_setup:Example`).
 
 1. Make sure the models, couplers and versions you want to use, are already available for `esm_master`
    to install them (``$ esm_master`` and check the list). If something is missing you will need to
-   add it following the instructions in :ref:`cookbook:Implementing a New Model`.
+   add it following the instructions in :ref:`cookbook:Implement a New Model`.
 
 2. Once everything you need is available to `esm_master`, you will need to create your own branch of
    `esm_tools`, following the steps 1-4 in :ref:`contributing:Contribution to esm_tools Package`.
 
-3. Then you will need to create a folder for your coupled setup inside ``esm_tools/configs/setups``
-   folder, and create a `yaml` file per version of your setup::
+3. Setups need two types of files: 1) **coupling files** containing information about model versions and
+   coupling changes, and 2) **setup files** containing the general information about the setup and the
+   model changes. In this step we focus on the creation of the **coupling files**.
+
+   a. Create a folder for your setup's couplings in ``esm_tools/configs/couplings``, and inside this
+      folder add a folder per coupling::
+
+       $ mkdir esm_tools/configs/couplings/<your_setup>
```

Comment:
I think there is no folder for the setup within the `couplings` folder. All the coupling folders go directly into `esm_tools/configs/couplings/`.

Line 242: `$ mkdir esm_tools/configs/couplings/<your_setup>` not needed.
Line 243: `$ cd esm_tools/configs/couplings/<your_setup>` can be
`$ cd esm_tools/configs/couplings/`

Reply:
You are totally right, thanks for the heads-up.